.travis.yml

```diff
@@ -21,3 +21,13 @@ install:
 script:
   - make test/flake8
   - make test
+
+deploy:
+  provider: pypi
```

Comment:
Might need to add a `distributions` key. I think by default it just builds a tarball, but it looks like you also provide a wheel.

https://github.com/ansible/galaxy_ng/blob/master/.travis.yml#L45

Reply:
Yes, thanks, my pre-release test missed the wheel, added it in.